dashboard/test/controllers/survey_results_controller_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class SurveyResultsControllerTest < ActionController::TestCase
+
+  setup do
+    @teacher = create(:teacher)
+  end
+
+  test "post survey results" do
+    sign_in @teacher
+    post :create, {survey: {survey2016_ethnicity_asian: 22}, format: :json}
+
+    survey_result = SurveyResult.where(user: @teacher).first
+    assert survey_result
+    assert survey_result["properties"]["survey2016_ethnicity_asian"] == 22
+  end
+
```

Comment:
creating the controller won't do anything interesting -- this looks fine since it is testing the behavior of your controller.

What would you assert about a created controller?